certif/app/controllers/authenticated/team/list/members.js

```diff
@@ -0,0 +1,14 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+
+export default class AuthenticatedTeamListMembersController extends Controller {
+  @service currentUser;
+  @service store;
+
+  @action
+  async leaveCertificationCenter() {
+    this.currentUser.currentCertificationCenterMembership.deleteRecord();
+    await this.currentUser.currentCertificationCenterMembership.save();
```

Comment:
**Suggestion** : 
Utiliser la méthode [destroyRecord](https://guides.emberjs.com/v4.12.0/models/creating-updating-and-deleting-records/#toc_deleting-records) qui traite la suppression et la persistance de la donnée en même temps.

```suggestion
    await this.currentUser.currentCertificationCenterMembership.destroyRecord();
```

Reply:
Corrigé sur ce fixup https://github.com/1024pix/pix/pull/7705/commits/b11faec15e2857eb4f57661616d7749858e0ee32